public/js/index.js

```diff
@@ -594,6 +594,7 @@ function initWikiForm() {
                         function (data) {
                             preview.innerHTML = '<div class="markdown">' + data + '</div>';
                             emojify.run($('.editor-preview')[0]);
+                            $('.editor-preview').autolink();
```

Comment:
@bkcsoft the context? You mean, the web page content? I've looked it up, googled it and found no reference of any penalty. What I did find is that Google does not always run JS. Yet, I bet Google crawler is smart enough to do autolinking itself. And even if it isn't the standard Markdown links should work flawlessly, we only lose unnecessary standard extensions. If anyone is willing to implement autolinking server-side after merging this PR, I would be glad to hear it.